File: src/main/java/um/tesoreria/core/hexagonal/personas/persona/infrastructure/web/dto/PersonaDto.java
package um.tesoreria.core.hexagonal.personas.persona.infrastructure.web.dto;

import java.math.BigDecimal;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PersonaDto {

    private BigDecimal personaId;
    private Integer documentoId;

    @Builder.Default
    private String apellido = "";

    @Builder.Default
    private String nombre = "";
}

File: src/main/java/um/tesoreria/core/kotlin/model/dto/PersonaDto.kt
package um.tesoreria.core.kotlin.model.dto

import java.math.BigDecimal

data class PersonaDto(

    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var apellido: String? = "",
    var nombre: String? = ""

)
	